### PR TITLE
Support for deployment in a multiple region

### PR DESCRIPTION
**Pull request by patrick-239**
*Issue #52 

*Description of changes:*
CNF templates, lambdas and Jupyter notebook updated to support a deployment in a multiple region

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by patrick-239** [comment]
Updated according to reconditions above.

File: stage.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ BUCKET=$1
 S3PATH=$2
 
 # remove this line if you want to keep the objects private in your S3 bucket
```
Comment (BastLeblanc) [inline stage.sh]: if we make it commented by default, I would change the comment to "uncomment this line [...]"

File: workshop/1-Personalization/1.1-Personalize.ipynb [hunk 2]
```diff
@@ -26,7 +26,7 @@
    "source": [
     "### Import Dependencies and Setup Boto3 Python Clients\n",
     "\n",
-    "Throughout this workshop we will need access to some common libraries and clients for connecting to AWS services."
+    "Throughout this workshop we will need access to some common libraries and clients for connecting to AWS services. We also have to retrive Uid from a SageMaker notebook instance tag."
```
Comment (BastLeblanc) [inline workshop/1-Personalization/1.1-Personalize.ipynb:29]: typo: retrieve

File: workshop/1-Personalization/1.2-Personalize-Cleanup.ipynb [hunk 3]
```diff
@@ -17,7 +17,7 @@
    "source": [
     "## Import Dependencies\n",
     "\n",
-    "To get started, let's import the dependencies we'll need for this notebook."
+    "To get started, let's import the dependencies we'll need for this notebook. We also have to retrive Uid from a SageMaker notebook instance tag."
```
Comment (BastLeblanc) [inline workshop/1-Personalization/1.2-Personalize-Cleanup.ipynb:20]: retrieve

File: workshop/4-Messaging/4.1-Pinpoint.ipynb [hunk 4]
```diff
@@ -160,7 +160,7 @@
    "source": [
     "### Import Dependencies\n",
     "\n",
-    "Next, let's import the dependencies we'll need for this notebook."
+    "Next, let's import the dependencies we'll need for this notebook. We also have to retrive Uid from a SageMaker notebook instance tag."
```
Comment (BastLeblanc) [inline workshop/4-Messaging/4.1-Pinpoint.ipynb:163]: retrieve

**Review by BastLeblanc — COMMENTED** [review]
small comments inline.

on the long name risk, can we estimate the longest resource name, how close are we from the limit? looking at it quickly with the modifications we have some good margins?

File: stage.sh (hunk 1 above)
Comment (patrick-239) [reply stage.sh]: This change is overwritten and will not pass into result commit.

**Comment by patrick-239** [comment]
Typos corrected

**Pull request merged**